src/test/java/org/cactoos/text/SubTest.java

```diff
@@ -34,6 +37,18 @@
  */
 final class SubTest {
 
+    @Test
+    void acceptsCharSequence() {
+        final Func<CharSequence, Integer> halfLength = new FuncOf<>(
+            sequence -> sequence.length() / 2
+        );
+        new Assertion<>(
+            "Can't cut a text with start",
```

Comment:
@DmitryBarskov please phrase it with "Must..." like all the other tests in cactoos :)